test/configurations/arm64/gyptest-arm64.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2012 Google Inc. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+Verifies build of an executable for ARM64 configurations.
+"""
+
+import TestGyp
+
+import sys
+
+formats = ['msvs']
+if sys.platform == 'win32':
+  formats += ['ninja']
+test = TestGyp.TestGyp(formats=formats)
+
+test.run_gyp('configurations.gyp')
+test.set_configuration('Debug|ARM64')
```

Comment:
On VS2017 I'm getting `The specified solution configuration "Debug|ARM64|x64" is invalid.`

Reply:
okay, could you send me how I can reproduce this?

Reply:
Note that VS2019 isn't out of beta yet, so it's possible that this is a VS bug.

Reply:
```cmd
python gyptest.py -v -f msvs test/configurations/arm64/gyptest-arm64.py
```

Reply:
I think it's a regression I introduced. Chasing this down.

Reply:
Solved.